Source/Scene/MapboxImageryProvider.js

```diff
@@ -0,0 +1,324 @@
+/*global define*/
+define([
+        '../Core/Credit',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/MapboxApi',
+        './UrlTemplateImageryProvider'
+    ], function(
+        Credit,
+        defaultValue,
+        defined,
+        defineProperties,
+        DeveloperError,
+        MapboxApi,
+        UrlTemplateImageryProvider) {
+    "use strict";
+
+    var trailingSlashRegex = /\/$/;
+    var defaultCredit1 = new Credit('© Mapbox © OpenStreetMap', undefined, 'https://www.mapbox.com/about/maps/');
+    var defaultCredit2 = new Credit('Improve this map', undefined, 'https://www.mapbox.com/map-feedback/');
```

Comment:
Do we need to provide the `Improve this map` link?  Seems a little weird in the context of Cesium.  If the answer is yet, it is kind of off to have it be a separate credit, but I guess we need that in order to have 2 links?  To bad credit text can't just be html.